docs/.vuepress/enhanceApp.js

```diff
@@ -1,6 +1,38 @@
-import landcycle from './scripts/Landcycle';
+import axios from 'axios';
 
 export default async ({ Vue }) => {
-  // Hydration class to parse linked data in JSON schemas
-  Vue.prototype.$landcycle = landcycle;
+  await axios
```

Comment:
Can we convert this to a function, since we have (almost) the same block of code 3x?

Reply:
I would say no, regardless a http call is isolated and individual. We can't make three calls at the same time and get three response back combined or otherwise. There's ways to reformat it with Promise.all but still, each calls expects different results